### ISSUE TITLE
autorestart-containers isn't working with wildcard domains

### ISSUE THREAD

**Issue by VoliKoN** [adferrand/docker-letsencrypt-dns#10]
Hello,
Everything works flawlessly for me without the autorestart-containers, once i added it i get weird errors.

domains.conf:
`*.home.example.com home.example.com autorestart-containers=nginx`

output (replaced my domain with home.example.com):
```
certbot    | *.home.example.com_autorestart-containers: added process group
certbot    | ERROR: /etc/letsencrypt/archive/*.home.example.com directory is missing.
certbot    | 2018-04-10 13:36:42,044 INFO exited:  *.home.example.com_autorestart-containers (exit status 1; not expected)

```

**Comment by adferrand** [comment]
Indeed, it is due to the fact that for wildcard certificates, the names of generated files does not match the primary domain name of the applied certificate (eg. *.example.com).

I will fix this tomorrow.

**Comment by adferrand** [comment]
Hello, I published a new version (2.0.1) which fixes this issue.

**Comment by VoliKoN** [comment]
I think it's working for me, Thank You!
